test/commonTests.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-env mocha */
+/* global expect */
+
+import nock from 'nock'
+
+export function notifiesWithUsageMessageForDashH() {
+  const {lgJWT, lgUser} = this
+  return this.invoke(['-h'], this.notify, {lgJWT, lgUser})
+    .then(() => {
+      expect(this.notifications[0]).to.match(/Usage:/)
+    })
+}
+
+export function notifiesWithUsageHintForInvalidArgs(invalidArgv) {
```

Comment:
I intentionally made it so that people could change the name / description of the test in contexts where that makes sense (e.g. invalid arguments). And for consistency, I wanted all of the helpers to be the same.